### PR TITLE
Feature 6766/New App for getting environment data within electron

### PR DESCRIPTION
**Pull request by PhotoNomad0**
#### Describe what your pull request addresses (Please include relevant issue numbers):
- New App for getting environment data when running within electron, other module, tools, and unit tests.

#### Please include detailed Test instructions for your pull request:
- checkout tc branch `bugfix-mcleanb-6766.2`
- Do `rm -rf ./node_modules && npm ci && npm run load-apps && npm start`
- wA, tN and tW tools should work and display properly. Particularly check lexicon popups.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+sudo: false
+language: node_js
+node_js:
+  - 6
```
Comment (mannycolon) [inline .travis.yml]: Why version 6?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+os:
+  - linux
+before_install:
+  - npm i -g npm@5.1.0
```
Comment (mannycolon) [inline .travis.yml]: ibid

File: .eslintrc [hunk 3]
```diff
@@ -0,0 +1,53 @@
+{
```
Comment (mannycolon) [inline .eslintrc:1]: I would suggest copying the eslint rules that we created for tC minus the react stuff

File: README.md [hunk 4]
```diff
@@ -1 +1,2 @@
-# tc-electron-env
\ No newline at end of file
+# tc-electron-env
```
Comment (mannycolon) [inline README.md:1]: Should we add an usage section on how to use this module?

**Review by mannycolon — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (PhotoNomad0) [reply .travis.yml]: Oops, just copied most files from word-aligner.

**Review by mannycolon — APPROVED** [review]

**Pull request merged**